recipes/netcdf-scm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: netcdf-scm
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/n/netcdf-scm/netcdf-scm-{{ version }}.tar.gz
+  sha256: 9a9e92982ea95ebbf12fc03abcc1659127e9bd7c8446841fd9dc39aa878f7ab4
+
+build:
+  number: 0
+  noarch: python
```

Comment:
@znicholls after looking at the source it seems we can make this package `noarch`. That means it will build faster and we don't have to rebuild it often. The trick is to constraint the python in the `host`/`run` requirements.

Reply:
If there were a sensible place to document that trick, it could help others.

Reply:
It is documented: https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
Ah ok. The fact that you can build noarch packages was clear. What wasn't obvious was that I could put a constraint on python in `host`/`run` sections, I thought I could only restrict package versions there. Having said that, I'm not sure if there's a way to make that more obvious in the docs. At some point people need to have a minimum amount of knowledge, which I don't have, or be given help by generous people like you, cheers :D